strawberryfields/_dev/configuration.py

```diff
@@ -54,14 +54,21 @@
 
     [api]
     # Options for the Strawberry Fields Cloud API
-    authentatication_token = "071cdcce-9241-4965-93af-4a4dbc739135"
+    authentication_token = "071cdcce-9241-4965-93af-4a4dbc739135"
     hostname = "localhost"
     use_ssl = true
 
 Summary of options
 ------------------
 
-todo
+SF_API_USE_SSL:
+    Whether to use SSL or not when connecting to the API. True or False.
+SF_API_HOSTNAME:
+    The hostname of the server to connect to. Defaults to localhost. Must be one of the allowed
+    hosts.
+SF_API_AUTHENTICATION_TOKEN:
+    The authentication token to use when connecting to the API. Will be sent with every request in
+    the header.
```

Comment:
Perhaps it's worth modifying this to specify the configuration file subsection (`[api]`) and lower case config keys (`use_ssl`, `hostname`, `authentication_token`), as well as the environment variables.

Reply:
@josh146 are you referring to something other than the above (lines 55-59)

Reply:
Oh, I meant more so that there are two ways the user can specify the options. Either as environment variables (`SF_API_AUTHENTICATION_TOKEN`) or in the config file, where it looks slightly different (no `SF_` prefix, `API` is the config section, and everything is lowercase):

```toml
[api]
authentication_token = 450984390845fgdsljkfsdg98324
```

So I was wondering if there was a way of including both environment variable key and config key in the glossary, maybe like

```rest
``SF_API_AUTHENTICATION_TOKEN`` (env variable) or ``authentication_token`` (file):
```
^ this looks a bit unwieldy, there is probably a better way of writing this.

Reply:
We could add a blurb explaining the conventions, e.g.:

```
...
The names of the environment variables are uppercased keys in the configuration file prefixed with
``SF_API_``. 

SF_API_USE_SSL:
    Whether to use SSL or not when connecting to the API. True or False.
...
```
Thoughts?